ruff.schema.json

```diff
@@ -1396,7 +1395,6 @@
         "C415",
         "C416",
         "C417",
-        "C9",
```

Comment:
So these work on the CLI (based on my testing), what's the rationale for excluding them?

Reply:
They only continue to work because I added redirects, see the "Update JSON schema" commit.

They are no longer generated by `cargo dev generate-json-schema` since these are partial linter prefixes ... I'd say we should consider them deprecated.

Reply:
Is this not the McCabe-equivalent of (e.g.) `--select B` for `flake8-bugbear`?

Reply:
(As an aside, would be nice to just remove these conflicts at some point.)

Reply:
> Is this not the McCabe-equivalent of (e.g.) --select B for flake8-bugbear?

No as per `#[prefix = "C90"] McCabe` in `registry.rs` the equivalent is C90.

Reply:
Note that this is also consistent with the upstream mccabe:

https://github.com/PyCQA/mccabe/blob/master/setup.py#L38